.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,21 @@
+name: CI-UnitTests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+            pip install django==3.2
```

Comment:
Can this be configured as a matrix so we can cover different django version, or add additional steps for each django version?

Reply:
Yeah, that would be the next step. To run the different versions of python and django.

I think how I would like to implement this. Would be a sort of multi step. When someone pushs an initial PR it runs the unittests for the oldest version first. If that passes and someone does an initial review of the code. It would trigger running the matrix. 

I think this makes sense, to save resources. There is no point in running the test matrix for something that is not yet finished.